### PR TITLE
Apply Ktlint plugin on project

### PR DESCRIPTION
**Pull request by jaredsburrows**
@jaredrummler 
- Apply Ktlint plugin `8.0.0`
- Update Kotlin to `1.3.31` for compatibility
- Format Kotlin code

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -41,21 +41,24 @@ buildscript {
 
   repositories {
     google()
-    jcenter()
```
Comment (jaredrummler) [inline build.gradle]: Should jcenter be removed? Should it be placed under `gradlePluginPortal`?

Comment (jaredsburrows) [reply build.gradle]: `gradlePluginPortal()` pull dependencies from `mavenCentral()`, `jcenter()` and from gradle plugins site.

**Comment by jaredsburrows** [comment]
Fixed the conflicts.

**Comment by jaredsburrows** [comment]
@jaredrummler Once this is merged, I will go through an manually fix the other Kotlin formatting stuff based on https://ktlint.github.io/, https://developer.android.com/kotlin/style-guide and https://kotlinlang.org/docs/reference/coding-conventions.html.

**Comment by jaredsburrows** [comment]
Any updates?

**Pull request merged**